controller/server/server.ml

```diff
@@ -52,8 +51,16 @@ let main debug port =
   (* Connect with ConnMan *)
   let%lwt connman = Connman.Manager.connect () in
 
+  let%lwt proxy = Connman_proxy.get () in
```

Comment:
Instead of relying on this external python script, we could get this with:

```ocaml
let%lwt proxy =
  Connman.Manager.get_services connman
  >|= List.find_map (fun s ->
    if s.Connman.Service.autoconnect then
      Option.bind s.Connman.Service.proxy Proxy.validate
    else
      None)
in
```

The advantage of using an external tool is to share the logic between the controller and the kiosk, but as a counterargument, we also have part of the logic duplicated in `gui.ml`.

Reply:
We’re now getting the proxy directly in Ocaml, as proposed above.